packages/grafana-ui/src/components/SingleStatShared/SingleStatBaseOptions.ts

```diff
@@ -331,3 +384,33 @@ export function migrateOldThresholds(thresholds?: any[]): Threshold[] | undefine
 export function convertOldAngularValueMapping(panel: any): ValueMapping[] {
   return convertOldAngularValueMappings(panel);
 }
+
+interface GraphOptions {
+  xaxis: {
+    mode: 'series' | 'time' | 'histogram';
+    values?: string[];
+  };
+  legend: {
+    show: boolean;
+    alignAsTable: boolean;
+    rightSide: boolean;
+    values: boolean;
+    min?: boolean;
+    max?: boolean;
+    avg?: boolean;
+    current?: boolean;
+    total?: boolean;
+    sideWidth?: number;
+  };
+}
+
+function getReducersFromLegend(obj: Record<string, unknown>): string[] {
+  const ids: string[] = [];
+  for (const key in obj) {
+    const r = fieldReducers.getIfExists(key);
```

Comment:
Nit: I prefer explicit naming when it comes to variables. I like `reducer` instead of `r`, just for clarity. Again, just a nit. 😁